src/services/codefixes/importFixes.ts

```diff
@@ -356,6 +357,24 @@ function createImportAdderWorker(sourceFile: SourceFile | FutureSourceFile, prog
         }
     }
 
+    function addImportForExternalModuleSymbol(symbol: Symbol, originalSymbol: Symbol, isValidTypeOnlyUseSite: boolean) {
+        const useRequire = shouldUseRequire(sourceFile, program);
+        if (originalSymbol.valueDeclaration) {

```

Comment:
What is this check for? When does an external module symbol not have a valueDeclaration?